org.scantailor.ScanTailor.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "org.scantailor.ScanTailor",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "scantailor",
+  "rename-appdata-file": "scantailor.appdata.xml",
+  "rename-desktop-file": "scantailor.desktop",
+  "rename-icon": "scantailor",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--filesystem=home"
+  ],
+  "modules": [
+    "shared-modules/qt4/qt4-4.8.7-minimal.json",
+    {
+      "name": "boost",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+          "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "./bootstrap.sh --prefix=/app",
```

Comment:
does it require the whole things from boost? you can build just the required modules